docs/accessibility/index.md

```diff
@@ -4,9 +4,26 @@ sidebar_position: 6
 
 Accessibility
 =======================
-Accessibility (WCAG 2-1, AA conformance) is a main concern of Joomla. This section describes how to develop accessible extensions for Joomla
+Accessibility is a main concern of Joomla!. 
+Find here Joomla's accessibility Statement https://www.joomla.org/accessibility-statement.html.
+
+The Joomla Core generates HTML sites which are WCAG 2.1 AA compliant. 
```

Comment:
I think its just as important to stress that Joomla itself is an accessible authoring tool not just that it can generate accessible sites.

Reply:
We have two parts - the manual is for developers, the documentation is a better place for teaching. 
But there will be many changes until this can be merged, so feel free to add suggestions

Reply:
yes - we have developers with accessibility needs and its important that we stress that joomla is accessible to use unlike other cms which only consider the output